src/gstreamer/gstkvssink.cpp

```diff
@@ -1609,6 +1622,18 @@ gst_kvs_sink_change_state(GstElement *element, GstStateChange transition) {
         case GST_STATE_CHANGE_READY_TO_PAUSED:
             gst_collect_pads_start (kvssink->collect);
             break;
+        default:
+            break;
+    }
+
+    // Parent class transition
+    ret = GST_ELEMENT_CLASS (parent_class)->change_state(element, transition);
```

Comment:
suggest adding  the return status and also the (parent) element and transition as debug logs..